framework/src/utils/MultiMooseEnum.C

```diff
@@ -57,143 +72,231 @@ MultiMooseEnum::operator!=(const MultiMooseEnum & value) const
   return !(*this == value);
 }
 
+bool
+MultiMooseEnum::isValueSet(const std::string & value) const
+{
+  return std::find_if(_current_values.begin(),
+                      _current_values.end(),
+                      [&value](const MooseEnumItem & item)
+                      { return item == value; }) != _current_values.end();
+}
+
 bool
 MultiMooseEnum::contains(const std::string & value) const
 {
-  return std::find_if(_current.begin(),
-                      _current.end(),
+  mooseDeprecated("MultiMooseEnum::contains is deprecated, use MultiMooseEnum::isValueSet");
```

Comment:
I m not sure we want to disallow the option to template on MultiMooseEnum with the same templates as std containers

Reply:
Would it be acceptable to add a `mooseInfo` statement clarifying that `contains` only operates on values the variable is set to, not possible values that the variable can be set to? We could also only have the info message active for `devel` and `dbg` builds, or maybe `dbg` builds only.

Reply:
I dont see why we would want to both:
- enable users to template with MultiMooseEnum just like a std container
- discourage them from doing so because the names aren't great

I think the new names are great and I'd just start using them over the old ones